packages/pigeon/example/app/android/app/src/main/java/io/flutter/plugins/Messages.java

```diff
@@ -55,6 +39,28 @@ protected static ArrayList<Object> wrapError(@NonNull Throwable exception) {
     return errorList;
   }
 
+  @NonNull
+  protected static FlutterError createConnectionError(@NonNull String channelName) {
+    return new FlutterError(
+        "channel-error", "Unable to establish connection on channel: " + channelName + ".", "");
+  }
+
+  /** Error class for passing custom error details to Flutter via a thrown PlatformException. */
+  public static class FlutterError extends RuntimeException {
```

Comment:
Why did this move to after methods? The old ordering seems clearer.

(Not a big deal if it's hard to change for structural generator reasons.)